### PR TITLE
fix margins from scattered css.

### PR DESCRIPTION
**Pull request by youngfreezyVA**
fixes margin issues with search bar on mobile:
https://github.com/department-of-veterans-affairs/va.gov-team/issues/16227

### CODE REVIEW

**Comment by youngfreezyVA** [comment]
how it looks now: Screen Shot 2020-12-03 at 1 45 34 PM

File: src/applications/resources-and-support/components/SearchBar.jsx [hunk 1]
```diff
@@ -29,7 +29,7 @@ export default function SearchBar({
   };
 
   return (
-    <div className="vads-u-border-top--2px vads-u-border-color--gray-light vads-u-padding-y--3 vads-u-padding-x--1">
+    <div className="medium-screen:vads-u-border-top--2px small-screen:vads-u-border-bottom--0px vads-u-border-color--gray-light vads-u-padding-y--3 vads-u-padding-x--1">
```
Comment (kelsonic) [inline src/applications/resources-and-support/components/SearchBar.jsx:32]: So we typically have mobile as default styles, then desktop styles prefixed with `medium-screen:`, so for example if you wanted the border bottom to be 0 on mobile but 1px on desktop, you would do `vads-u-border-bottom--0px medium-screen:vads-u-border-bottom--1px` 🙂

Comment (kelsonic) [reply src/applications/resources-and-support/components/SearchBar.jsx:32]: I almost never use `small-screen:` because of ^^

Comment (youngfreezyVA) [reply src/applications/resources-and-support/components/SearchBar.jsx:32]: ah yes, mobile first!

**Review by kelsonic — APPROVED** [review]
Nice! 🎉 Before you merge, could you check what it looks like on both mobile and desktop just to make sure? Thanks for picking this up! 🔥

**Review by ncksllvn — APPROVED** [review]

File: src/applications/resources-and-support/components/SearchBar.jsx [hunk 2]
```diff
@@ -46,7 +46,7 @@ export default function SearchBar({
   };
 
   return (
-    <div className="vads-u-border-top--2px vads-u-border-color--gray-light vads-u-padding-y--3 vads-u-padding-x--1">
+    <div className="vads-u-border-bottom--0px medium-screen:vads-u-border-top--2px vads-u-border-color--gray-light vads-u-padding-y--3 vads-u-padding-x--1">
```
Comment (U-DON) [inline src/applications/resources-and-support/components/SearchBar.jsx:49]: You want `vads-u-border-bottom--0`, without the `px`. [Reference](https://design.va.gov/utilities/border)

**Review by U-DON — APPROVED** [review]
There's a minor issue with a potentially invalid class name, but otherwise looks good.

**Pull request merged**